### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
- Moved the "This is not a romhack" part to the beginning of the readme.
- Pushed the features list to the bottom, as it may have had an effect on burying other points of the readme.
- Added a "Should I use this or vanilla pokeemerald for my hack?" section near the top.
- Added link to documentation page.
- Potentially standarize the name of "Expansion Senate"? (We can discuss this if needed)
- Other redaction improvements

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,54 @@
...
-
-pokeemerald-expansion is a decomp hack base project based off pret's [pokeemerald](https://github.com/pret/pokeemerald) decompilation project. It's recommended that any new projects that plan on using it, to clone this repository instead of pret's vanilla repository, as we regurlarly incorporate pret's documentation changes. This is ***NOT*** a standalone romhack, and as such, most features will be unavailable and/or unbalanced if played as is.
+## Should I use this or vanilla pokeemerald for my hack?
+The main advantage of using vanilla pokeemerald as a base is being able to link with other official GBA Pokémon games for battles and trading, pokeemerald-expansion can battle and trade with itself out of the box. If you don't mind losing full vanilla compatiblitity, we recommend using pokeemerald-expansion. Otherwise, use pret's pokeemerald. You'll still receive documentation improvements from pret, as we regurlarly incorporate pret's documentation changes.
 
 If you use pokeemerald-expansion in your hack, please add RHH (Rom Hacking Hideout) to your credits list. Optionally, you can list the version used, so it can help players know what features to expect.
```
Comment (Pawkkie) [inline README.md:8]: I think this should get a heading, something like

## Using pokeemerald-expansion

That goes from "If you use pokeemerald-expansion in your hack" to the start of the "Important:" PSA. I think we should include the sentence
`Please follow the instructions in `INSTALL.md` to get pokeemerald-expansion set up on your machine.`
at the end of this section, as per #5643 (and then I'll close that PR).

**Review by Pawkkie — COMMENTED** [review]
Left some comments, I think this looks good otherwise, all subjective of course :)

File: README.md [hunk 2]
```diff
@@ -1,17 +1,54 @@
 # pokeemerald-expansion
 
-### Important: DO NOT use GitHub's "Download Zip" option. Using this option will not download the commit history required to update your expansion version or merge other feature branches. Instead, please read [this guide](https://github.com/Pawkkie/Team-Aquas-Asset-Repo/wiki/The-Basics-of-GitHub) to learn how to fork the repository and clone locally from there.
+pokeemerald-expansion is ***NOT a playable romhack. It is a romhack base*** based off pret's [pokeemerald](https://github.com/pret/pokeemerald) decompilation project. It has multiple features available to romhackers so that they can create their own games, but it's not meant to be played on its own.
```
[the remark this answers is not in the document]
Comment (AsparagusEduardo) [reply README.md:3]: I don't disagree. I put the "not a hack" part first mainly out of spite. I can put it as the second sentence, but making it all caps, so it's still the first thing that skimmers see.

File: README.md (hunk 1 above)
Comment (Pawkkie) [reply README.md:8]: Ignore this feedback, PR'd separately in #5720

**Comment by hedara90** [comment]
Conflicts from #5720 (I think)

**Comment by AsparagusEduardo** [comment]
Fixed conflicts and adjusted order of first paragraph.

**Review by hedara90 — APPROVED** [review]

**Pull request merged**